packages/kotti-ui/source/kotti-popover/KtPopover.vue

```diff
@@ -19,13 +26,24 @@
 import { createPopper } from '@popperjs/core'
 import { mixin as clickaway } from 'vue-clickaway'
 
+const optionIsValid = (option) =>
+	(!option.disabled || typeof option.disabled === 'boolean') &&
+	option.onClick &&
+	typeof option.onClick === 'function' &&
+	option.text &&
```

Comment:
If this is a new API, this should be `label` to align with `KtField*Select`

Reply:
ok correct